git-secrets

```diff
@@ -69,7 +69,9 @@ scan() {
 }
 
 scan_history() {
-  local to_scan=$(git rev-list --all)
+  local patterns=$(load_patterns | tr ' ' '|')
```

Comment:
Problem: git log does not support multiple regular expressions.
Solution: Combine all patterns with `|`.